freight/api/serializer/deploy.py

```diff
@@ -54,14 +63,19 @@ def get_attrs(self, item_list):
     def serialize(self, item, attrs):
         app = attrs["app"]
         task = attrs["task"]
+        repo = attrs["repo"]
+
+        vcs_remote = vcsremote.get_by_repo(repo)
 
         return {
             "id": str(item.id),
             "name": f"{app.name}/{item.environment}#{item.number}",
             "app": {"id": str(app.id), "name": app.name},
+            "remote": {"name": vcs_remote.hostname, "url": vcs_remote.repo_url},
             "user": serialize(attrs["user"]),
             "environment": item.environment,
             "sha": task.sha,
+            "sha_url": vcs_remote.get_commit_url(task.sha),
```

Comment:
Just FYI: it's nontrivial for Freight to fetch sentry sha for getsentry in the current state of things. I assume your primary goal here is same repo.

Reply:
Yeah unfortunately we would need to build out more advanced functionality to do that.

It won't be terribly hard since freight already has a clone of the repo and can enumerate commits tho. So we could probably build something that handles special lookup of commits due to the way the getsentry bot makes the commit message.

Reply:
Yes, but it's a bit of a stretch since it's in getsentry-deploy... it would be ideal if there would be functionality for a script to export some JSON for freight to read, similar to GHA.